### PR TITLE
Created Wall of Love page

### PR DESCRIPTION
**Pull request by alfonsogoberjr**
Made the [Wall of Love](https://github.com/styled-components/styled-components-website/issues/17) page with [this list](https://github.com/styled-components/styled-components.github.io/issues/3) of tweets. Accessible at `/wall-of-love`, there is also a button on the homepage and a clickable logo on the Wall of Love to take you back to `/`. 

Feel free to recommend any design changes, I simply used the pink gradient background from the homepage and included the footer for consistency. 

Cheers

### CODE REVIEW

**Comment by mxstbr** [comment]
This is great! 😍 

Four nitpicky comments:

1. Can we hide the tweet context on some of those tweets, especially these ones?
  screen shot 2017-06-11 at 13 33 28
2. Let's remove the tweet by Kent C. Dodds, he's now the creator of glamorous so it'd be awkward to advertise him saying how great s-c is :blush:
3. Can we server-side render the tweets? Is there a module that will let us do that by any chance?
4. I don't like the Button on the homepage, that feels too... prominent to me, but I don't really have a better idea of where to link to it :confused: @philpl?

**Comment by philpl** [comment]
@alfonsogoberjr I have similar suggestions, as @mxstbr's :smile:

> Can we hide the tweet context on some of those tweets, especially these ones?
> Can we server-side render the tweets? Is there a module that will let us do that by any chance?

Optimally we'd want our tweets to be hardcoded or fetched in `getInitialProps` and then displayed using some custom components, since that'll give us some freedom to alter the design :wink:

> I don't like the Button on the homepage, that feels too... prominent to me, but I don't really have a better idea of where to link to it 

We'll have to add a navigation menu bar at the top now, since I think that's the cleanest way to deal with the future pages that we'll add. If you'd like to do this, @alfonsogoberjr, I can create a quick design for you :)

Another thing is, once we have a navigation menu, the header of this page should not include the text, logo and so on at the top. Also the content should be displayed on a white background. The footer should also look the same as on the homepage.

Basically the content should be presented in the same way, as it is presented on the homepage. With the same title and text style. The navigation bar is sth that we can also add in a separate PR first :smile:

**Comment by alfonsogoberjr** [comment]
@mxstbr @philpl Happy to implement all changes, but I'd definitely prefer to use a mockup as a single source of truth for what is desired here. I'll wait until that's available before making any updates.

**Comment by philpl** [comment]
@alfonsogoberjr I'll try to get a design ready for you, but I think the menu-bar should be done separately maybe. Not sure if you meant that you're interested in doing this task as well?

**Comment by alfonsogoberjr** [comment]
@philpl I agree the menu-bar should be done separately, but either way I prefer to do front-end work based on mockups, so whichever you want to make them for first, I'm happy to work on.

**Comment by alfonsogoberjr** [comment]
Btw thanks for all the feedback! 😄

**Comment by morajabi** [comment]
Are you working on it?

**Comment by mckernanin** [comment]
@morajabi @alfonsogoberjr need any help here?

**Comment by alfonsogoberjr** [comment]
@mckernanin @morajabi Still blocked pending mockups from @kitten or any other styled-components  site designer
